.github/workflows/deploy-dev-ecs.yml

```diff
@@ -1,9 +1,12 @@
 on:
   push:
-    branches-ignore:
-      - "integration"
+    branches:
       - "master"
-      - "meta"
+      - "integration"
```

Comment:
@casthewiz Are there situations where you foresee we'd want the dev deploy to run prior to a PR being opened? My feeling is that folks use the dev builds to confirm that their work is - vaguely - working prior to opening up a PR, but I'm not sure that's actually an accurate assessment.

If it *is* an accurate assessment, we may want to revisit the changes to this file.

Reply:
So the pipeline definitely needs followup changes IMO. I'd opt for have the build/test pipelines only run for commits that are part of a PR. I think running the pipeline for any (non-PR) branch can get out of control.